src/codegen/cfg.rs

```diff
@@ -189,6 +189,14 @@ pub enum Instr {
     /// be removed. We only have this so we can pass evm code through sema/codegen, which is used
     /// by the language server and the ethereum solidity tests.
     Unimplemented { reachable: bool },
+    /// This instruction serves to track account accesses through 'tx.accounts.my_account'
+    /// on Solana, and has no emit implementation. It is exchanged by the proper
+    /// Expression::Subscript at solana_accounts/account_management.rs
+    AccountAccess {
```

Comment:
We really need  #1354

Reply:
How does this help here?
I made a hack in codegen, but you suggested features flags for LLVM. Would they encompass codegen too?